### PR TITLE
Introduce poke

### PR DESCRIPTION
**Pull request by phritz**
This PR is a couple of very minor edits on top of https://github.com/rocicorp/replicache/pull/686. The actual work was done by @aboodman .

### CODE REVIEW

**Comment by phritz** [comment]
(feel free to merge if checks pass, i have to go adult)

File: src/sync/pull.ts [hunk 1]
```diff
@@ -101,29 +100,50 @@ export async function beginPull(
     };
   }
 
+  const syncHead = await handlePullResponse(lc, store, baseCookie, response);
+  if (syncHead === null) {
+    throw new Error('Overlapping sync JsLogInfo');
```
Comment (arv) [inline src/sync/pull.ts:105]: JsLogInfo should not be needed any more

File: src/replicache.test.ts [hunk 2]
```diff
@@ -1656,6 +1656,83 @@ testWithBothStores('HTTP status push', async () => {
   expect(okCalled).to.equal(true);
 });
 
+testWithBothStores('poke', async () => {
+  // TODO(MP) test:
+  // - when we queue a poke and it matches, we update the snapshot
+  // - rebase still works
+  // - when the cookie doesn't match, it doesn't apply, but later when the cookie matches it does
+  // - per-client timing
+  const rep = await replicacheForTesting('pull', {
```
Comment (arv) [inline src/replicache.test.ts]: s/pull/poke/

**Review by arv — APPROVED** [review]
LGTM

**Pull request merged**